src/SIPdump.h

```diff
@@ -361,8 +361,9 @@ int find_value(const char *value, const char *buffer, char *outbuf,
 		}
 	}
 
-	strncpy(outbuf, tempbuf, outbuf_len);
-	outbuf[outbuf_len] = 0;
+	strncpy(outbuf, tempbuf, outbuf_len - 1);
+	if (outbuf_len > 0)
+		outbuf[outbuf_len - 1] = 0;
```

Comment:
So outbuf_len can be zero at this point? How will the preceding strncpy cope with that situation? Its size argument is `size_t`...